### ISSUE TITLE
Railcraft Rockcrusher addRecipie crash

### ISSUE THREAD

**Issue by egeis** [jaredlll08/ModTweaker2#150]
I added the following line to my scripts for railcraft changes.  On /mt reload no error messages are presented.  Examining the any item in the output list using NEI Integration however causes a crash.

This error also occurs during processing in the rock crusher.  The LooseRock is placed in the output fine but when the small ore is randomly added the client crashes.  The error is identical to the one posted below.

mods.railcraft.RockCrusher.addRecipe(<terrafirmacraft:Gravel:0>, false, false,
  [<terrafirmacraft:item.LooseRock:0>,
  <terrafirmacraft:item.Small Ore:4>,
  <terrafirmacraft:item.Small Ore:1>,
  <terrafirmacraft:item.Small Ore:5>,
  <terrafirmacraft:item.Small Ore:28>],
  [0.5,0.05,0.05,0.05,0.05]);

Pastebin crash log: http://pastebin.com/YjbxEkte

**Comment by Voidi** [comment]
`[terrafirmacraft:item.LooseRock:0,
,
,
,
],`
shouldn't there be entrys between the commas?

**Comment by egeis** [comment]
thats funny when i pasted there were entries, let me try again.

Comments are because ive disabled the recipie for now.
//mods.railcraft.RockCrusher.addRecipe(<terrafirmacraft:Gravel:0>, false, false,
//  [<terrafirmacraft:item.LooseRock:0>,
//  <terrafirmacraft:item.Small Ore:4>,
//  <terrafirmacraft:item.Small Ore:1>,
//  <terrafirmacraft:item.Small Ore:5>,
//  <terrafirmacraft:item.Small Ore:28>],
//  [0.5,0.05,0.05,0.05,0.05]);

**Comment by egeis** [comment]
its not pasting properly with the entries, they are "terrafirmacraft:item.Small Ore:#" where # is 4,1,5,28 in that order.

**Comment by egeis** [comment]
The error i believe is with how the space is handled in the name.  Unfortunately some mods using spaces in their names....  All names without spaces in terrafirmacraft work fine in the rock crusher.

**Comment by Voidi** [comment]
Yes that's very likely the problem.
Had the same Problem by some fluids (but a problem from minetweaker not modtweaker).

**Comment by jaredlll08** [comment]
Minetweaker issue. also mods are NEVER meant to have a space in unloc names. ALSO NEVER USE /MT RELOAD!

**Comment by Yulife** [comment]
They are not meant to but they can do it. Also, it is an issue on MT's end, not the mod's.

**Comment by jaredlll08** [comment]
Uh no. A mod should NEVER have a dpace in the unlocalized name. That is a
minecraft rule, try /give @a <item with a space in name>
It WILL fail.

**Comment by Voidi** [comment]
Hm
The given item uses indeed whitespace, according to minetweaker and NEI.
Because i had a similar issue a year ago i asked the devs http://terrafirmacraft.com/f/topic/6433-internal-item-names-with-whitespaces/.

And when uses this item in vanilla workbench recipe it works.
Confusing.

**Comment by jaredlll08** [comment]
Yep that is normal, well you mean in code or script? Code =normal script =
something is messed

**Comment by Zixxl** [comment]
Hmm, can't reproduce this bug.

Added the command:
```mods.railcraft.RockCrusher.addRecipe(<terrafirmacraft:Gravel:0>, false, false, [<terrafirmacraft:item.Small Ore:4>], [1.0]);```

Log:
```INFO: [ModTweaker2] Adding 1 Railcraft Rock Crusher Recipe(s) for <terrafirmacraft:Gravel>```

Working Crusher:
http://imgur.com/hZPQjP0
